synapse/models/infotech.py

```diff
@@ -321,12 +321,15 @@ def getModelDefs(self):
                 }),
             ),
             'types': (
+
                 ('it:hostname', ('str', {'strip': True, 'lower': True}), {
-                    'doc': 'The name of a host or system.',
-                }),
+                    'doc': 'The name of a host or system.'}),
+
+
                 ('it:host', ('guid', {}), {
-                    'doc': 'A GUID that represents a host or system.'
-                }),
+                    'interfaces': ('inet:service:object',),
+                    'doc': 'A GUID that represents a host or system.'}),
```

Comment:
The docstrings for the inet:service:object interface props are a bit confusing when applied to a "host" (and make most sense when the host is virtual). Tried to clarify below. Was unable to make further suggestions on any it:host props directly (since they weren't changed).